shareaholic.info.yml

```diff
@@ -0,0 +1,14 @@
+type: module
+name: "Shareaholic - Share Buttons, Analytics, Related Posts"
+description: "Get people sharing your content, keep them on your site longer, grow your fans, monetize, and know who's reading your content. Powered by <a href='https://www.shareaholic.com/' target='_blank'>Shareaholic</a>."
+
+configure: shareaholic.settings
+
+tags:
+  - shareaholic
+
+# Information added by Drupal.org packaging script on 2019-07-22
```

Comment:
Lines 10-14 are added by Drupal.org automatically. I don't believe we need to manually add this and it will change each time a release is tagged.

Reply:
Done.